src/http-api.ts

```diff
@@ -5,25 +5,35 @@ import responseTime from 'response-time';
 
 import cors from 'cors';
 import express from 'express';
-import { register, Histogram } from 'prom-client';
+import { register, Histogram, Counter } from 'prom-client';
+import morgan from 'morgan';
 
 import { v4 as uuidv4 } from 'uuid';
 
 import { jsonRpcApi } from './json-rpc-api.js';
+import { JsonRpcRequest } from './lib/json-rpc.js';
 import { createJsonRpcErrorResponse, JsonRpcErrorCodes } from './lib/json-rpc.js';
 
 export class HttpApi {
+  requestCounter = new Counter({
+    name       : 'dwn_requests_total',
+    help       : 'all dwn requests processed',
+    labelNames : ['method', 'status', 'error'],
+  });
+
   api: Express;
   dwn: Dwn;
 
   constructor(dwn: Dwn) {
     this.api = express();
     this.dwn = dwn;
 
+    this.api.use(morgan('tiny')); // custom log formats are fairly easy, if we want to use a different format: https://expressjs.com/en/resources/middleware/morgan.html
+
     const responseHistogram = new Histogram({
       name       : 'http_response',
       help       : 'response histogram',
-      buckets    : [50, 250, 500, 750, 1000],
+      buckets    : [0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10],
```

Comment:
bucket numbers are in _seconds_, as far as I know we are not expecting DWN requests to take up to 16 minutes. replacement values are the defaults. I left the list in rather than remove the variable entirely and let the library use it's defaults so that we can be reminded to tweak it in the future. I'm not super familiar with how to make the server work but when i ran the web5-js tests against it nothing was under 0.25 seconds